### PR TITLE
feat: Add `.github` repository

### PR DESCRIPTION
**Pull request by fuzzypixelz**
We're planning to use this to store community health files such as `SECURITY` and `CODE_OF_CONDUCT` in a single source of truth. An example would be the rust-lang organization where `SECURITY.md` is stored in https://github.com/rust-lang/.github and is accessible from https://github.com/rust-lang/rust?tab=security-ov-file.

@netomi What do you think of such a setup?

### CODE REVIEW

**Comment by netomi** [comment]
Thats a standard setup of many projects and good practice imho.

**Comment by netomi** [comment]
so all LGTM, if you added project lead as reviewer and get approval you can merge yourself.

**Comment by kydos** [comment]
+1

**Review by netomi — APPROVED** [review]

**Pull request merged**
